src/components/routine/RoutineForm.tsx

```
@@ -72,83 +45,134 @@ const RoutineForm = ({ nickname, mateNickname }: RoutineFormProps) => {
+              }
+
+              onChange(e);
+            }}
+          />
+        )}
+      />
+      <FormItem
+        name="endDate"
+        className="flex flex-col gap-2 mt-5"
+        label="루틴 종료 날짜"
+        render={({ value, name, onChange, form, setValue }) => (
```

ESLint

[error] 'form' is already declared in the upper scope on line 30 column 9.

(@typescript-eslint/no-shadow)